pkg/flavor/flavor.go

```diff
@@ -15,8 +14,10 @@ import (
 )
 
 type Methods interface {
-	FindProfile(vm *virtv1.VirtualMachine) (*flavorv1alpha1.VirtualMachineFlavorProfile, error)
-	ApplyToVmi(field *k8sfield.Path, profile *flavorv1alpha1.VirtualMachineFlavorProfile, vmiSpec *virtv1.VirtualMachineInstanceSpec) Conflicts
+	FindAndApplyFlavor(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) (Conflicts, error)
+	FindFlavor(vm *virtv1.VirtualMachine) (*flavorv1alpha1.VirtualMachineFlavor, error)
+	FindClusterFlavor(vm *virtv1.VirtualMachine) (*flavorv1alpha1.VirtualMachineClusterFlavor, error)
```

Comment:
These 2 methods `FindFlavor` and `FindClusterFlavor` are not used anywhere (only in unit tests). Do you plan to use them in a future PR?

Reply:
I think I did at the time but yeah you're right they don't need to be public, I'll change this.